tests/core/test_broker.py

```diff
@@ -310,6 +311,23 @@ async def test_no_pika_logs_if_no_debug_mode(caplog: LogCaptureFixture):
         for record in caplog.records
     )
 
+    broker = PikaEventBroker(
+        "host", "username", "password", retry_delay_in_seconds=1, connection_attempts=1
+    )
```

Comment:
I'm not exactly sure what these new lines are for 🤔 If you added by mistake, could you please remove?

Reply:
@jppgomes This also needs to be resolved.